components/exchange/AddToDeckModal.js

```diff
@@ -0,0 +1,145 @@
+import React, { useContext, useState } from 'react'
+import styled from 'styled-components'
+import { CheckboxChecked } from 'styled-icons/icomoon/CheckboxChecked'
+import { Error } from 'styled-icons/boxicons-solid/Error'
+import { BlackButton } from '../ui-fragments/Button'
+import DeckBox from '../browse-decks/DeckBox'
+import { DeckListsContext } from '../DeckListsContextProvider'
+import LoggedInView from '../ui-fragments/LoggedInView'
+import Modal from '../ui-fragments/Modal'
+import screenSizes from '../../utils/screen-sizes'
+import { RelativeSpinner } from '../ui-fragments/Spinner'
+import Deck from '../deck/model/Deck'
+
+const ModalContent = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  flex: 1;
+  margin-bottom: 40px;
+
+  @media screen and (max-width: ${screenSizes.smallPhone.max}) {
+    display: block;
+    justify-content: center;
+    align-items: center;
```

Comment:
The cards are a bit misaligned on mobile. This will make them take the full width.

```suggestion
    align-items: center;
     
    a {
      width: 100%;
    }
```